recipes.mdwn

```diff
@@ -41,3 +41,4 @@ to improve your Awesome setup.
 
 * [[Swap Monitor Snippet|recipes/xrandr]]
 * [Poppin' - Pop over applications](https://github.com/raksooo/poppin)
+* [[Stop unfocused clients|recipes/stop_unfocused]]
```

Comment:
Perhaps `SIGSTOP unfocused clients`? That might be clearer (at least to everyone knowing about SIGSTOP).